### PR TITLE
ClonePlayer function for SpawnHandler

### PR DESCRIPTION
**Pull request by Aranclanos**
### Purpose
Adds a static function for SpawnHandler called ClonePlayer() and merges it with CreateGhost()

### Open Questions and Pre-Merge TODOs

- [X]  This fix is tested on the same branch it is PR'ed to.
- [X]  I correctly commented my code
- [X]  My code is indented with tabs and not spaces
- [X]  This PR does not include any unnecessary .meta, .prefab or .unity (scene) changes
- [X]  This PR does not bring up any new compile errors
- [X]  This PR has been tested in editor
- [ ]  This PR has been tested in multiplayer (with 2 clients and late joiners, if applicable)

### Notes:
I tried to merge it with CreatePlayer() but the spawn objects dont have a registertile and are a bit of a headache to work with

### CODE REVIEW

File: UnityProject/Assets/Scripts/Managers/NetworkManagement/SpawnHandler.cs [hunk 1]
```diff
@@ -72,18 +79,18 @@ public static void TransferPlayer(NetworkConnection conn, short playerController
 			playerScript.characterSettings = characterSettings;}
 	}
 
-	private static GameObject CreateGhost(GameObject oldBody)
+	private static GameObject CreateMob(JobType jobType, GameObject spawnSpot, GameObject mobPrefab)
 	{
-		GameObject ghostPrefab = CustomNetworkManager.Instance.ghostPrefab;
-		Vector3 spawnPosition = oldBody.GetComponent<ObjectBehaviour>().AssumedLocation();
```
Comment (fomalsd) [inline UnityProject/Assets/Scripts/Managers/NetworkManagement/SpawnHandler.cs:78]: I'm not sure about replacement of AssumedLocation with WorldPosition, it'll probably cause issues when someone dies inside a locker

Comment (fomalsd) [reply UnityProject/Assets/Scripts/Managers/NetworkManagement/SpawnHandler.cs:78]: yep
yep, it causes issues

**Review by fomalsd — CHANGES_REQUESTED** [review]
spawn position must still be AssumedLocation

**Comment by Aranclanos** [comment]
something went really wrong

**Pull request closed without merging**